php-nginx/build-scripts/build_php70.sh

```diff
@@ -40,14 +40,17 @@ curl -SL "https://pecl.php.net/get/mailparse" -o mailparse.tar.gz
 tar -zxf mailparse.tar.gz -C ${PHP_SRC}/ext/mailparse --strip-components=1
 rm mailparse.tar.gz
 
-# TODO: Use the stable version of apcu from pecl once available.
-# With the 5.1.2 source from the PECL, it doesn'r recognize the
-# --enable-apcu option.
-# mkdir -p ${PHP_SRC}/ext/apcu
-# curl -SL "https://pecl.php.net/get/apcu" -o apcu.tar.gz
-# tar -zxf apcu.tar.gz -C ${PHP_SRC}/ext/apcu --strip-components=1
-# rm apcu.tar.gz
-git clone https://github.com/krakjoe/apcu.git ${PHP_SRC}/ext/apcu
+# APCu
+mkdir -p ${PHP_SRC}/ext/apcu
+curl -SL "https://pecl.php.net/get/apcu-5.1.3.tgz" -o apcu.tar.gz
```

Comment:
Actually I used the specific version on purpose, so the container always has a specified state. Do you want me to change it?